engine/src/btc/rpc.rs

```diff
@@ -86,15 +86,10 @@ impl BtcRpcClient {
 					match get_bitcoin_network(&client, &endpoint).await {
 						Ok(network) if network == expected_btc_network => break,
 						Ok(network) => {
-							error!(
```

Comment:
fyi you can use r#"..." to do this without having to put everything on one line. I believe, haven't used it myself before.

Reply:
They're called raw string literals.

Reply:
I think raw string literals is the opposite, it will always print the formatting, including any escapes i.e. it prints the raw message. Here I could use `\` to escape the newline in the message (not using raw string literals). I'll do that, a little nicer.